### ISSUE TITLE
project treble

### ISSUE THREAD

**Issue by mahmoudim** [OnePlusOSS/android_kernel_oneplus_msm8998#15]
will you enable project treble for android 8.0 on OP5 or OP5t ?

**Comment by always9988** [comment]
No, both OP5 and OP5t  launch to market  on android 7.x. The layout can't support project treble.

**Comment by jumoog** [comment]
you can.. if you have a reserved partition like Sony has

**Comment by ghost** [comment]
We (the community) would like to ask Oneplus to support Project Treble in OOS. Petition link: http://chn.ge/2BaA6jY

**Comment by ghost** [comment]
@leonfish77101
